### PR TITLE
Styling syntax as a code block

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/visual-basic/programming-guide/language-features/procedures/operator-procedures.md [hunk 1]
```diff
@@ -37,13 +37,15 @@ An operator procedure is a series of Visual Basic statements that define the beh
   
 ## Declaration Syntax  
  The syntax for declaring an operator procedure is as follows:  
+ 
+ ```vb 
+ Public Shared [Widening | Narrowing] Operator *operatorsymbol* ( *operand1* [,  *operand2* ]) As *datatype*
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/procedures/operator-procedures.md]: The \* here were used to make the names italic. By using a code block, we should probably remove those:

```suggestion
 Public Shared [Widening | Narrowing] Operator operatorsymbol ( operand1 [,  operand2 ]) As datatype  
```

This is how the change renders right now:
image

**Review by mairaw — CHANGES_REQUESTED** [review]
I like this change @Youssef1313! I just left a suggestion for an improvement on the formatting due to the change from text to code block. Please take a look!

**Comment by Youssef1313** [comment]
> 
> 
> I like this change @Youssef1313! I just left a suggestion for an improvement on the formatting due to the change from text to code block. Please take a look!

@mairaw , Yes, it seems a lot better. I've committed your change.

**Review by mairaw — APPROVED** [review]
Looks great! Thanks @Youssef1313!

**Pull request merged**